dev/tools/dartdoc.dart

```diff
@@ -173,6 +173,7 @@ Future<void> main(List<String> arguments) async {
       'analyzer',
       'args',
       'barback',
+      'cli_util',
```

Comment:
We decided this should be reverted from https://github.com/flutter/flutter/pull/99037 since `cli_util` is still a dependency.

Reply:
:-(